.github/workflows/builds.yml

```diff
@@ -44,5 +44,13 @@ jobs:
       - name: Install only test dependencies
         run: poetry install --no-root --extras "ray" --with dev
 
+      - name: Setup Julia
```

Comment:
will this workflow also run the julia tests? (is there any flag you have to add to the pytest section?)